index.html

```diff
@@ -36,6 +36,321 @@ <h5>LET’S CONNECT</h5>
+
+
+              <div class="article3div">
+
+
+                  <h3 class="texttitle font1">Tonic</h3>
+
+
+                  <ul>
+
+
+                    <li class="texttitle font2">CANOPY</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">Back End Dev</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">2015</li>
+                  </ul>
+
+
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+
+                  <ul class="float_box1">
+
+
+                      <li class="background1"><a class="texttitle font4" href="#">html</a></li>
+                      <li class="background1"><a class="texttitle font4" href="#">css</a></li>
+                      <li class="background1"><a class="texttitle font4" href="#">javaScript</a></li>
+
+
+                  </ul>
+
+
+                  <button type="submit"><a class="font5 texttitle" href="#">See Project</a></button>
+
+
+              </div>
+
+
+          </article>
+
+
+          <article class="article4">
+
+
+              <figure>
+
+<img src="images/Snapshoot Portfolio (3).png" alt="Portfolio4">
+
+
+              </figure>
+
+
+              <div class="article4div">
+
+
+                  <h3 class="font1">Multi-Post Stories</h3>
+
+
+                  <ul>
+
+
+                    <li class="texttitle font2">CANOPY</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">Back End Dev</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">2015</li>
+
+
+                  </ul>
+
+
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+
+                  <ul class="float_box1">
+
+
+                      <li class="background1"><a class="texttitle font4" href="#">html</a></li>
+                      <li class="background1"><a class="texttitle font4" href="#">css</a></li>
+                      <li class="background1"><a class="texttitle font4" href="#">javaScript</a></li>
+
+
+                  </ul>
+
+
+                  <button type="submit"><a class="font5 texttitle" href="#">See Project</a></button>
+
+
+              </div>
+
+
+          </article>
+
+
+
+
+      </section>
+
+
+      <section class="about_self_section">
+
+
+          <div>
+
+
+              <h3 class="font1">About Myself</h3>
+<br>
+
+              <p>Hello I'm a software developer! I can help you build a product, feature or website Look through some of
+                  my work and experience! if you like what you see and have a project you need coded, don't hesistate to
+                  contact me.</p>
+
+
+              <h5>LET'S CONNECT</h5>
```

Comment:
- Kindly, uses the font defined for the template and stick as much as posible with font weight, style....👌👍